core/new-gui/src/app/workspace/component/menu/menu.component.ts

```diff
@@ -470,7 +470,7 @@ export class MenuComponent implements OnInit {
             : "" +
               this.datePipe.transform(
                 this.workflowActionService.getWorkflowMetadata().creationTime,
-                "MM/dd/yyyy HH:mm:ss zzz",
```

Comment:
Can we keep it?

Reply:
I removed the timezone to save some space, and it aligns with the other timestamps in our system. We didn't show the timezone in the other place.

Reply:
The other place not show zzz was a compromise... but I'm ok as long as all places are aligned and no confusion